src/binaryen-c.cpp

```diff
@@ -1616,6 +1616,17 @@ double BinaryenConstGetValueF64(BinaryenExpressionRef expr) {
   assert(expression->is<Const>());
   return static_cast<Const*>(expression)->value.getf64();
 }
+const uint8_t* BinaryenConstGetValueV128(BinaryenExpressionRef expr) {
+  if (tracing) {
+    std::cout << "  BinaryenConstGetValueV128(expressions[" << expressions[expr] << "]);\n";
+  }
+
+  auto* expression = (Expression*)expr;
+  assert(expression->is<Const>());
+  void* buf = malloc(16);
+  memcpy(buf, static_cast<Const*>(expression)->value.getv128().data(), 16);
```

Comment:
Seems this passes the checker now, still this one is a bit unfortunate in that `.getv128()` already `malloc`s and there's certainly a better way that avoids one of the allocations. What would I do if I were clever? :)